Sorting/MergeSort.php

```diff
@@ -8,49 +8,45 @@
  */
 function mergeSort(array $arr)
 {
-    if (count($arr) <= 1)
-    {
+    if (count($arr) <= 1) {
```

Comment:
We try to follow the PER style guide, which states that opening braces belong on their own line

https://www.php-fig.org/per/coding-style/#44-methods-and-functions